src/frontend/src/app/core/state/proposal/proposal.mapper.ts

```diff
@@ -54,6 +54,57 @@ export function mapOpenProposalListResponse(
   return proposalsList;
 }
 
+export function mapClosedProposalListResponse(
+  apiResponseList: ProposalResponse[],
+): Proposal[] {
+  const proposalsList: Proposal[] = [];
+
+  for (const proposalResponse of apiResponseList) {
+    const proposalObject: Proposal = {
+      id: proposalResponse.proposal.nervous_system.network.id,
+      title: proposalResponse.proposal.title,
+      topic: getProposalTopic(
+        proposalResponse.proposal.nervous_system.network.topic,
+      ),
+      type: 'unknown',
+      state: ProposalState.Completed,
+      proposedAt: new Date(proposalResponse.proposal.proposed_at),
+      proposedBy: proposalResponse.proposal.proposed_by,
+      reviewCompletedAt: proposalResponse.proposal.review_completed_at[0]
```

Comment:
We could make a reusable function for this because we will encounter it multiple times